database/database.go

```diff
@@ -77,20 +77,34 @@ func CreateNewBoardDB(name string, active bool) bool {
 	if active {
```

Comment:
We also need to check if the name of the new board already exists as a name in the db

Reply:
Completed